### ISSUE TITLE
Not working properly with ${execPath} to subdirectory within VSCode portable

### ISSUE THREAD

**Issue by carloscadux** [sketchbuch/vsc-workspace-sidebar#43]
Hi. 
As shown in the snapshot, (1) although the extension recognizes ${execPath} variable, listing all existing workspaces, (2) when I click on the workspace name, (3) there is the error "the path... does not exist..."

Here is a reference about ${execPath}
[portability - How to reference in settings a path to subdirectory within VSCode portable (relative to 'Code.exe')? - Stack Overflow](https://stackoverflow.com/questions/71289334/how-to-reference-in-settings-a-path-to-subdirectory-within-vscode-portable-rela/71300411#71300411)

Thank you!

![image](https://user-images.githubusercontent.com/7339389/157271997-f9156d62-1bc7-4400-880c-1f7c37896371.png)

**Comment by sketchbuch** [comment]
Hi, what is exactly execPath? I'm surprised it shows the workspaces still. Is this some build in var that system level functions in node are aware of? What does this var point to?

**Comment by carloscadux** [comment]
Hey @sketchbuch, 
I'm ignorant about VSCode development.
In practice, I use this variable exclusively in settings of extensions, to refer to subdirectories paths within VSCode portable (relative to 'Code.exe').
For instance (screenshot above), I added $`{execPath}\..\data_workspaces` in your extension settings and it listed the workspaces I had stored in that folder.
In the thread I referred to in my initial post I tried to assemble some pieces of knowledge I gathered on the topic.
This is the [Visual Studio Code Variables Reference](https://code.visualstudio.com/docs/editor/variables-reference), in which execPath is mentioned, but very briefly and with a shallow explanation. 
In the following thread, a developer gave me an explanation that is more technical
[Possible to set 'Data Location' path to subdirectory within VSCode portable](https://github.com/oleg-shilo/Favorites.vscode/issues/33):
> Though keep in mind that VSCode is failing to expand variables in the values of the config settings. Nothing really can be done about it. It's just a limitation of VSCode.
> 
> Though in this case I simply replace `${execPath}` with the return value of the `process.exePath`.
> 
> Thus your desired path should be:
> 
> ```
> ${execPath}\..\data_cadu
> ```
> 
> Note extra `..` as `exePath` is not the dir but the exe file.

This other thread is not about `exePath` itself, but refers also to VSCode portability of an extension:
[If `'Data Location'` is set `<default>`, the `favorites.user` folder could be created within portable path (e.g. '..\globalStorage\favorites.user`) · Issue #37 · oleg-shilo/Favorites.vscode](https://github.com/oleg-shilo/Favorites.vscode/issues/37)

I hope this helps 👍
